pset-functions.js

```diff
@@ -147,6 +205,39 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 4:')
 
 // Add your code below this line
+//create a function called doubleLetters with word parameter
+// function doubleLetters(word){
+//   //loop through the word
+//   for(i = 0; i <= word.length; i++) {
+//     //loop through the word again (behind it)
+//         for(var j = i+1; j <= word.length; j++) {
+//           //compare results, if results match, it has double letters
+//             if(word[j] == word[i]) {
+//                 return true;
+//             }
+//         }
+//     }
+//     return false;
+// }
+
+//refactored using arrow function
+const doubleLetters = (word) => {
+  for(i = 0; i <= word.length; i++) {
+    //loop through the word again (behind it)
+        for(var j = i+1; j <= word.length; j++) {
```

Comment:
Matt, great way of making use of the nested loop's iteration count. Although sometimes difficult to avoid, nested loops can cause code to take longer than you intend to run. Alternatively, you can use one for loop and take advantage of the fact that we are comparing current index with the one after (so index + 1). To see what that looks like, [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L225) is a sample solution.